keyboards/crkbd/keymaps/antosha417/keymap.c

```diff
@@ -0,0 +1,401 @@
+// Copyright 2021 Anton Kavalkou (@antosha417)
+// SPDX-License-Identifier: GPL-2.0-or-later
+#include QMK_KEYBOARD_H
+#include <sendstring_dvorak.h>
+
+typedef union {
+  uint32_t raw;
+  struct {
+    bool is_macos :1;
+  };
+} user_config;
+
+user_config config;
```

Comment:
Just a heads up, you can probably use the magic config for this.  Aka "keycode_config",  for the alt-gui or control-gui swap settings.

Reply:
Thank you for the suggestion. 
I do not want to swap gui key with any other. For me control-gui swap would make most sense but I use vim a lot. And in vim there all kinds of control + something shortcuts. I do not want them to be different on linux and macos.

Reply:
Okay, makes sense.  Figured it was worth mentioning, at least.